### ISSUE TITLE
Biconda installation fails

### ISSUE THREAD

**Issue by sinanugur** [sinanugur/MirMachine#7]
mamba can install but conda install fails when installing.

**Comment by RickiG** [comment]
@sinanugur I am experiencing this with v0.2.11.1 when installing through Bioconda or Forge.
First it would just time out in `solving environment`. After a Conda update I could get it to the error message:

```ResolvePackageNotFound:
  - python=3.1```

However, I don't see any dependencies in your .yml or requirements file for python3.1 and there is no python3.1 available on bioconductor.
Hope this helps narrow it down 👍

**Comment by sinanugur** [comment]
That is new. Can you try installing in a clean environment like this:

`conda install -c bioconda -c conda-forge mirmachine`

I noticed, `conda-forge` must be added, otherwise it returns solving problems.

If that does not work, mamba can be another solution. Mamba resolves the environment faster.

```
conda install mamba
mamba install -c bioconda -c conda-forge mirmachine
```

**Comment by RickiG** [comment]
I still can get `conda install -c bioconda -c conda-forge mirmachine` to resolve (I'm on a Mac, maybe it Is architecture related) 

I did manage to. install the previous version v0.2.11 through conda. 

I just tried mamba, resolved v0.2.11.1 almost instantly - no problem!
